server/views/submit.ejs

```diff
@@ -1,17 +1,17 @@
 <%var title = 'Submit'%>
 
-<link rel="stylesheet" type="text/css" href="./css/submit.css">
+<!-- <link rel="stylesheet" type="text/css" href="./css/submit.css"> -->
 <script type="text/javascript" src="./js/submit.js"></script>
 
-<main class="container">
+<main class="col d-flex justify-content-center">
     <!-- Ticket Submission widget -->
-    <div class="card">
-        <form id="ticketForm">
+    <div class="card text-center align-item-center" style="margin-bottom: auto; margin-top: 5%">
+        <form id="ticketForm" style="width: 80vh">

```

Comment:
`width: 80vh` doesn't make too much sense because `vh` is relative to the height of the viewport. You probably wanted `vw` here.